README.md

```diff
@@ -85,10 +85,7 @@ therefore lacking these features:
 - [Some ELF linking features](https://github.com/ziglang/zig/issues/17749)
 - [Most COFF/PE linking features](https://github.com/ziglang/zig/issues/17751)
 - [Some WebAssembly linking features](https://github.com/ziglang/zig/issues/17750)
-- [Ability to output LLVM bitcode](https://github.com/ziglang/zig/issues/13265)
-- [Windows resource file compilation](https://github.com/ziglang/zig/issues/17752)
 - [Ability to create import libs from def files](https://github.com/ziglang/zig/issues/17807)
-- [Automatic importlib file generation for Windows DLLs](https://github.com/ziglang/zig/issues/17753)
```

Comment:
it still depends on the WriteImportLibrary function:

https://github.com/ziglang/zig/blob/7789e87230778b93fb9a584722dcc6d8a5b2030f/src/mingw.zig#L283